gamemodes/terrortown/entities/entities/ttt_cse_proj.lua

```diff
@@ -233,14 +229,13 @@ if CLIENT then
 
         tData:SetTitle(TryT("vis_name"))
 
-        if roleData.isPublicRole and roleData.isPolicingRole then
+        if ent:GetOriginator() == client then
             tData:SetSubtitle(ParT("target_pickup", { usekey = Key("+use", "USE") }))
-            tData:SetKeyBinding("+use")
         else
-            tData:SetSubtitle(TryT("vis_no_pickup"))
-            tData:AddIcon(roles.DETECTIVE.iconMaterial)
+            tData:SetSubtitle(TryT("entity_pickup_owner_only"))
         end
 
+        tData:SetKeyBinding("+use")
```

Comment:
now everbody can see the keybinding, could you please change it back so that only people that are able to pick it up see the binding?